src/webui/www/private/preferences_content.html

```diff
@@ -165,6 +165,10 @@
     <input type="checkbox" id="force_proxy_checkbox"/>
     <label for="force_proxy_checkbox">QBT_TR(Disable connections not supported by proxies)QBT_TR[CONTEXT=OptionsDialog]</label>
     </div>
+    <div class="formRow">
+    <input type="checkbox" id="proxy_only_for_torrents_checkbox"/>
+    <label for="proxy_only_for_torrents_checkbox">QBT_TR(Use proxy only for torrents)QBT_TR[CONTEXT=OptionsDialog]</label>
```

Comment:
Offtopic:
@Chocobo1, following your idea about moving WebUI into separate repo and the fact someone can develop alternative WebUI now, I suggest to use separate translations for WebUI.

Reply:
I'm not sure, that means twice work for translators & duplicate strings in binary.

Reply:
1. Alternative WebUI can be very different from qBittorrent GUI and have to use additional strings.
2. Even in official WebUI we need some additional strings (extra_translations.h, unless I'm wrong in file name)
3. When we change GUI it can be side effect now unless we change WebUI synchronously

I suggest to allow WebUI provide optional translation files in its source tree. It still can use any strings from GUI.

Reply:
IMO for alternative webUI,  we should provide them our translated strings (if possible), if they need additional ones, they provide it themselves. Or another way, can we just say we don't provide this functionality?
For official webUI, if we move extra_translations.h to the new repo, then transifex might need to add that repo too, @sledgehammer999 should agree with this.

Reply:
Well, we need to discuss this with the people concerned (e.g. @WolfganP). And, of course, we need separate Issue for it if we want to continue on.

Reply:
I think two different sets of strings and translation/internalization projects will be difficult to maintain in the long run. I would rather vote for using the core webUI strings as base, and then contain the extras in a pre-defined file (ie extra_translations.h or similar as proposed by @glassez) to mantain projects consistency.

Reply:
I am not sure if you have opened a new issue for this.
Here is my quick thought on this:
1. Predefine a path AND a filename where the .qm file needs to be located. Eg <webui root>/translations.qm
2. WebUI authors can themselves manage the creation of the relevant .ts file, get it translated, build the .qm file from it, and finally distribute it with their alt-web-ui package.
3. The qm file can hold more than 1 language at a time. It doesn't need to be one lang per .qm like we do.
4. The WebUI server, detects if there is a file at the predetermined location, loads the translations but does not attach them to the rest of the app. I think this is possible with Qt but I don't remember the exact procedure at the moment. And I think it is fair to make those translations have precedence over the translations provided by us for the same strings. (If the author wants to override some of our translations).
5. IIRC the only purpose of extra_translations.h is to make it possible for lupdate to pickup the translatable strings and put them inside the .ts file. But as long as somehow you provide a qm file that has translations for string A the explicit lookup will succeed and will fetch the translation.

Reply:
@sledgehammer999, it's about the way I thought of it.
But apparently I don't need to rush into it until we have requests for this feature.